docs/contents/user-docs/authorization.md

```diff
@@ -0,0 +1,145 @@
+# Authorization
+
+When we talk about authorization and access control for ACK, we need to discuss
+two different RBAC systems.
+
+Remember that [Kubernetes RBAC][0] governs a **Kubernetes** user's ability to
+read or write **Kubernetes resources**. In the case of ACK, this means that
+Kubernetes RBAC system controls the ability of a Kubernetes user to read or
+write different custom resources (CRs) exposed by ACK service controllers.
```

Comment:
"exposed by ACK service controllers" maybe say "that ACK service controllers use"

Reply:
done.